forest/src/cli/config.rs

```diff
@@ -4,19 +4,57 @@
 use forest_chain_sync::SyncConfig;
 use forest_libp2p::Libp2pConfig;
 use forest_networks::ChainConfig;
+use log::LevelFilter;
 use serde::{Deserialize, Serialize};
-use std::collections::HashSet;
 use std::path::PathBuf;
 use std::sync::Arc;
 
 use super::client::Client;
 
-#[derive(Serialize, Deserialize, Default, PartialEq, Eq)]
-pub struct LogConfig {
-    pub log_values: HashSet<LogValue>,
+#[derive(Serialize, Deserialize, PartialEq, Eq)]
+pub struct LogConfig(pub Vec<LogValue>);
+
+impl Default for LogConfig {
+    fn default() -> Self {
+        let underlying = vec![
+            LogValue {
```

Comment:
It would be nicer if you'd implement a custom constructor for `LogValue`, then we would get rid of lots of this boilerplate. A usage I would imagine:

```
let underlying = vec![
LogValue::new("libp2p_gossipsub", LevelFilter::Error),
LogValue::new("filecoin_proofs", LevelFilter::Warn),
...
];
```
What do you think?

Reply:
Implemented

Reply:
Thanks, it looks way better now!